lib/tag-finder.coffee

```diff
@@ -8,14 +8,14 @@ SelfClosingTags = require './self-closing-tags'
 module.exports =
 class TagFinder
   constructor: (@editor) ->
-    @tagPattern = /(<(\/?))([^\s>]+)([\s>]|$)/
-    @wordRegex = /[^>\r\n]*/
+    @tagPattern = /(<(\/?))([^/\s>]+)([^/>]*)(\/?)(>)/
```

Comment:
for reference, the test html I'm using:

    <TEST>
        <foo bar="/">
        </foo>
        <TEST/>
    </TEST>

so your suggestion does help with recognizing a tag like `<foo bar="/">`, but introduces a weird problem where, if I implement the fix on both `@tagPattern` and in `patternForTagName`, then putting the cursor on the last closing tag `</TEST>` doesn't highlight that pair (it highlights fine with the opening tag).

If I replace only the `@tagPattern` sequence, the program doesn't match the closing `</foo>` in the same manner as the closing `</TEST>`.

Looking into this problem now.